lib/services/comment_creator.rb

```diff
@@ -0,0 +1,17 @@
+class CommentCreator
+  def initialize(parsed_email)
+    @parsed_email = parsed_email
+  end
+
+  def run
```

Comment:
on the one hand I like how little this is doing. On the other hand, `inbound_mail_parser` is doing most of the work here so maybe it's not as helpful as I originally thought